test/honeylemon/MarketContractProxy.test.js

```diff
@@ -98,7 +98,7 @@ contract(
       _currentMri = new BigNumber(pc.getMRIDataForDay(0)).multipliedBy(
         new BigNumber(1e8)
       );
-      _expiration = Math.round(new Date().getTime() / 1000) + 3600 * 24 * 28;
+      _expiration = (await marketContractProxy.getTime()).toNumber() + 3600 * 24 * 28;
```

Comment:
I changed this through the tests so that they are not dependent on local time but rather use the contract time